### PR TITLE
Added a sweet alert notifier pop up

### PR DESCRIPTION
**Pull request by DerrickLegacy**

### CODE REVIEW

File: src/pages/AppraisalActivity/ActivityTableData/AddNewInitiativeDetailsModal.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Button from 'react-bootstrap/Button';
 import Form from 'react-bootstrap/Form';
 import Modal from 'react-bootstrap/Modal';
 
-function AddNewInitiativeDetailsModal({ measurableActivity, show, handleClose }) {
+function AddNewInitiativeDetailsModal({displyaSuccessMessage, measurableActivity, show, handleClose }) {
```
Comment (Nakatolynn) [inline src/pages/AppraisalActivity/ActivityTableData/AddNewInitiativeDetailsModal.jsx:6]: Change this to displaySuccessMessage

File: src/pages/AppraisalActivity/ActivityTableData/DetailPage.js [hunk 2]
```diff
@@ -5,23 +5,60 @@ import { FaArrowLeft } from 'react-icons/fa';
...
+      text: 'Record Saved Successfully',
+      icon: 'success',
+      showConfirmButton: false,
+      timerProgressBar:true,
+      timer: 2000
+    });
+  };
+
+  const handleSave = () => {
+    console.log('saved');
```
Comment (Nakatolynn) [inline src/pages/AppraisalActivity/ActivityTableData/DetailPage.js:53]: Remove this  Console.log please

File: src/pages/AppraisalActivity/ActivityTableData/DetailPage.js [hunk 3]
```diff
@@ -58,7 +95,7 @@ const DetailPage = () => {
             color="error"
             size="small"
             onClick={() => {
-              // Implement delete functionality
+              confirmDelete();
               console.log("Delete action for row:", row.original);
```
Comment (Nakatolynn) [inline src/pages/AppraisalActivity/ActivityTableData/DetailPage.js:99]: Derrick this console should be removed.

File: src/pages/Dashboard/Dashboard.js [hunk 4]
```diff
@@ -2,21 +2,38 @@ import React, { useState, useEffect } from "react";
...
 function Dashboard() {
   const [activeTab, setActiveTab] = useState("Q1");
   const [tabDisplay, setTabDisplay] = useState(true);
   const [quartileFilter, setQuartileFilter] = useState("Q1");
 
+  useEffect(() => {
+    const isFirstLoad = localStorage.getItem("isFirstLoad");
+    if (!isFirstLoad) {
+      Swal.fire({
+        position: 'top',
```
Comment (Nakatolynn) [inline src/pages/Dashboard/Dashboard.js:16]: position:'center'

Comment (mariotury) [reply src/pages/Dashboard/Dashboard.js]: @DerrickLegacy the welcome alert is showing every time the page reloads. correct that please.

File: src/pages/AppraisalActivity/ActivityTableData/DetailPage.js (hunk 2 above)
Comment (mariotury) [reply src/pages/AppraisalActivity/ActivityTableData/DetailPage.js]: @DerrickLegacy what you have done for the delete function with the sweetrAlerts, do the same for the edit function.

**Review by mariotury — CHANGES_REQUESTED** [review]
make changes to everything as stated in the comments. Also note that the appraisal details page is not as we agreed. Make the Measurable Activitiy stand out.
Screenshot 2024-07-31 100750

File: src/pages/Dashboard/Dashboard.js (hunk 4 above)
Comment (mariotury) [reply src/pages/Dashboard/Dashboard.js]: What value is it adding, again?

**Pull request closed without merging**